crates/csskit_proc_macro/src/def.rs

```diff
@@ -388,6 +389,11 @@ impl Def {
 				let ident = format_ident!("Literal{}", variant_str);
 				quote! { #ident }
 			},
+			Self::Combinator(ds, DefCombinatorStyle::Ordered) => {
```

Comment:
Just pick the first non optional def, and fallback to the first optional. Not sure if this will be elegant to use for 100% of the cases, but does look kinda nice now:

```
"<color>? auto"
Auto((Optional<Color>, Ident))
```